test/fixtures/rules/std_rule_cases/L028.yml

```diff
@@ -303,3 +303,10 @@ passes_tql_table_variable:
     rules:
       L028:
         single_table_references: qualified
+
+fail_but_dont_fix_templated_table_name:
```

Comment:
Should we add a similar qualified test?

Reply:
For this particular issue, the rule configuration (default vs consistent) shouldn't have an effect. Do you feel strongly about it?

Reply:
I added the test. Was just asking because I was on the phone and couldn't add tests earlier. 😛